site-src/v1alpha2/references/cross-namespace-references.md

```diff
@@ -0,0 +1,127 @@
+# Cross namespace references and ReferencePolicy
+
+### Terminology note
```

Comment:
This feels a bit out of place. Not sure how this would render with mkdocs, but maybe something like a blockquote or callout format with this content added in the Intro section right before/after "Referent" is mentioned for the first time.

Reply:
Yeah, I didn't like it there either, but couldn't think of the right place to put it. I'll see how we can do a blockquote or something more inline instead.

Reply:
I think I found a "Note" syntax that should work, I'll check the deploy preview.

Reply:
I added in an "admonitions" extension, that we can use with:

```
!!! admonition
    blockquote text is indented by four spaces
```

Reply:
This looks perfect! Thanks for adding it in